FTBQ_lang/1036-1048 【III - Mars Attacks!】.json

```
@@ -1,16 +1,16 @@
-	"ftbquests.chapter.iii_2.quests1.description": "Combining Elementalcraft Runes with an Ars Nouveau Element, some Manasteel and Manapowder will result into Botania Runes[--将元素工艺符文与新生魔艺元素结合,一些魔钢和魔力粉将生成植物魔法符文--]",
-	"ftbquests.chapter.iii_2.quests8.description": "salt into a chemical oxidizer gives you gaseous brine. Put that into a rotary condenstrator to make Brine (Liquid). [--将盐变成化学氧化剂,会给你气态盐水。放入旋转冷凝器中制作盐水(液态)。--]",
-	"ftbquests.chapter.iii_2.quests9.description": "Liquid Brine goes into an electrolytic seperator to give you sodium and chlorine gas. [--液态盐水进入电解静音器,会给你钠和氯气.--]",
-	"ftbquests.chapter.iii_2.quests10.description": "Combine silicon, Chlorine Gas and Polyethylene in a Pressurized Reaction Chamber to make printed silicon, the basis of all advanced circuits. You'll definitely want this process automated.[--在加压反应室中将硅、氯气和聚乙烯结合,以制作所有的高级电路基件(即纸质硅).你绝对需要这个自动化过程.--]",
-	"ftbquests.chapter.iii_2.quests16.description0": "&eLaserIO&r intelligently teleports items between inventories, without needing to have a path of pipes.[--&eLaserIO&r在物品栏之间智能传送物品,不需要有管道通道.--]",
-	"ftbquests.chapter.iii_2.quests16.description2": "The mod has a very good &ePatchouli&r book, which I suggest you read.[--mod有一个非常不错的&ePatchouli&r书,我建议你阅读.--]",
-	"ftbquests.chapter.iii_2.quests25.subtitle": "Let it grow, let it grow![--让它生长,让它生长!--]",
-	"ftbquests.chapter.iii_2.quests25.description0": "Crafting a &6Manasteel Mesh&r will make you more noticeable to magic users of the world![--制作一个&6魔钢筛网&r可以让你更看重世界的魔法使用者!--]",
-	"ftbquests.chapter.iii_2.quests25.description2": "Amethyst can be looted or traded from piglins. They can then be infused with elemental water magic to create springaline. [--紫水晶可以被猪灵抢夺或交易。然后它们可以灌注元素水魔法来制造春素。--]",
-	"ftbquests.chapter.iii_2.quests25.description4": "&6Amethyst Golems&r are very useful for farming and creating additional amethyt, check out the quest for them in the Ars Nouveau chapter![--&6紫晶傀儡&r对耕种和创造额外的紫水晶很有用,查看新生秘典章节中他们的任务!--]",
-	"ftbquests.chapter.iii_2.quests25.description6": "You'll then need to make your way to the moon to gather some Desh ore. By alloying the Springaline with Desh you can create a magically stable metal that and refine even more resources from mundane items.[--然后,你需要前往月球收集一些戴斯泰斯矿石。通过将斯普林亚灵与戴斯铁合金,你可以制作一个神奇稳定的金属,从普通物品中提炼更多的资源。--]"
+	"_comment_iii_2": "译者：Addyabbia  校对：",
+	"ftbquests.chapter.iii_2.title": "火星",
+	"ftbquests.chapter.iii_2.quests0.description": "&b符文祭坛&r是最早需要使用微缩合成制作的方块之一。请查看 JEI 如何放置方块，并丢入魔力钻石启动制作",
+	"ftbquests.chapter.iii_2.quests1.description": "将元素工艺符文、新生魔艺元素相、魔力钢和魔力尘放置于符文祭坛制作植物魔法符文",
+	"ftbquests.chapter.iii_2.quests8.description": "将盐放入化学氧化机，就会生产气态盐水。将其输入回旋式气液转换机中，制成液态盐水",
+	"ftbquests.chapter.iii_2.quests9.description": "液态盐水输入电解分离机，产生钠蒸汽和氯气",
+	"ftbquests.chapter.iii_2.quests10.description": "将硅、氯气和聚乙烯输入加压反应室制作硅板，这是制作更高级电路板的基础材料。您肯定希望这一过程能够自动化",
+	"ftbquests.chapter.iii_2.quests16.description0": "&eLaserIO&r能在库存之间智能的传输物品，不需要管道线缆",
```

```suggestion
	"ftbquests.chapter.iii_2.quests16.description0": "&eLaserIO&r能在容器之间智能的传输物品，不需要管道线缆",
```
inventories物品栏或者物品容器